common/perf-test-core/pom.xml

```diff
@@ -81,5 +81,10 @@
       <artifactId>azure-core-http-okhttp</artifactId>
       <version>1.11.17</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;dependency} -->
     </dependency>
+    <dependency>
```

Comment:
Two questions:

- Can we also include Vert.x as well? We need to start getting performance numbers for it as well.
- Do we create the uber JAR correctly where it includes all the HttpClientProviders in `META-INF`?

Both can be punted to another PR if needed.

Reply:
1. Sure, added vertx.
2. It does not need SPI to create clients since it knows the type and I'd rather keep it this way to simplify things, especially for perf tests which use this jar